brightray/brightray.gypi

```diff
@@ -271,13 +271,19 @@
               # can be removed at link time with --gc-sections.
               '-fdata-sections',
               '-ffunction-sections',
+              '-flto=thin',
```

Comment:
Do we enable ThinLTO on all arch ? Chromium seems to be doing it only for x86_64 (but yeah they don't ship any 32-bit versions) so we might be safe with that but how about arm ? We should probably have release build CI setup for this PR if thats possible, thoughts ? /cc @jkleinsc

Reply:
Yeah, I expected to iron out these things with the help of CI, but didn't realize it doesn't build the release configuration. Oh well, more work for me locally. :)